proto/platform/platform.proto

```diff
@@ -0,0 +1,17 @@
+syntax = "proto3";
+
+package platform;
+
+option go_package = "github.com/ava-labs/avalanchego/proto/pb/platform";
+
+// Platform defines the P-Chain APIs
+service Platform {
```

Comment:
This doesn't need to be included as part of this PR (as we already have tests through xsvm)... I can unstage this if we want.

Reply:
Removed